janus.c

```diff
@@ -3935,8 +3939,9 @@ json_t *janus_plugin_handle_sdp(janus_plugin_session *plugin_session, janus_plug
 				int rtx_ptype = ptype+1;
 				if(rtx_ptype > 127)
 					rtx_ptype = 96;
-				while(g_list_find(m->ptypes, GINT_TO_POINTER(rtx_ptype))
-						|| g_list_find(rtx_ptypes, GINT_TO_POINTER(rtx_ptype))) {
+				while(g_list_find(m->ptypes, GINT_TO_POINTER(rtx_ptype)) || g_list_find(rtx_ptypes, GINT_TO_POINTER(rtx_ptype)) ||
+					(m_audio_ptypes && g_list_find(m_audio_ptypes, GINT_TO_POINTER(rtx_ptype))) ||
+					(m_application_ptypes && g_list_find(m_application_ptypes, GINT_TO_POINTER(rtx_ptype)))) {
```

Comment:
Tiny visual nit: there should be an additional indent for the two lines above, as right now they're aligned with the action below.